rpc-jobs/install-jenkins-plugins.yml

```diff
@@ -0,0 +1,37 @@
+- job:
+    name: 'Install-Jenkins-Plugins'
+    description: Runs ansible playbook to install required Jenkins plugins
+    logrotate:
+        daysToKeep: 20
+    parameters:
+        - string:
+            name: JJB_REPO
+            default: https://github.com/rcbops/rpc-gating
+        - string:
+            name: JJB_BRANCH
+            default: master
+    scm:
+        - git:
+            url: $JJB_REPO
+            branches:
+                - $JJB_BRANCH
+            skip-tag: true
+            wipe-workspace: false
+    wrappers:
+      - credentials-binding:
+          - username-password-separated:
+             credential-id: service_account_jenkins_api_creds
+             username: JENKINS_USER
+             password: JENKINS_API_PASSWORD
+    builders:
+        - shell: |
+            scl enable python27 bash
```

Comment:
pipeline-steps/common.groovy has install_ansible() for installing ansible and venv_playbook() for running a playbook from ansible in a venv. Those functions set the right wrappers and env vars for colours etc.